web/components/SchedulePage.jsx

```diff
@@ -67,10 +56,130 @@ class SchedulePage extends React.Component {
         this.props.setSelectedDay(days[0])
     }
 
+    componentWillReceiveProps(nextProps) {
```

Comment:
Refactor this to use `componentDidUpdate` since `componentWillReceiveProps` will be deprecated soon.

https://stackoverflow.com/questions/51980977/why-is-componentwillreceiveprops-deprecated